app/src/main/java/com/duckduckgo/app/cta/ui/CtaViewModel.kt

```diff
@@ -334,7 +334,7 @@ class CtaViewModel @Inject constructor(
                 return null
             }
 
-            if (!canShowOnboardingDaxDialogCta()) {
+            if (areInContextDaxDialogsCompleted()) {
```

Comment:
I'd remove `canShowOnboardingDaxDialogCta()` method since it is no longer used anywhere.